src/qt/dcclient.cpp

```diff
@@ -27,6 +27,28 @@
 
 //##############################################################################
 
+namespace
+{
+    class QtPictureSetter
```

Comment:
Trivial, but could you please avoid indenting namespace contents? We don't do it elsewhere and consistency is nice.